src/Utils/Translate.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Translate.h"
+
+#include "SKSE/Translation.h"
+
+namespace Util
+{
+	std::string Translate(const std::string& key)

```

Comment:
define a string literal for this function e.g. t"" or something so we don't need to write the full function everytime.